src/pages/mentees.tsx

```diff
@@ -77,6 +78,14 @@ function MenteeTable({ users, refetch }: {
   users: User[],
   refetch: () => void
 }) {
+  const [menteeToYear, setMenteeToYear] = useState<Record<string, string>>({}); 
+  const updateMenteeYear = useCallback((id: string, year: string) => {
+    setMenteeToYear((currentMenteeToYear) => ({
```

Comment:
`currentMenteeToYear` => `current`. the smaller the scope of a variable, the shorter the variable name should be.